kubectl-plugin/pkg/cmd/delete/delete.go

```diff
@@ -0,0 +1,152 @@
+package kubectlraydelete
```

Comment:
`package delete` to be consistent with other packages?

Reply:
I did try to be consistent with the other packages but `delete` counts as a "predeclared identifier" when linting.

Reply:
`delete` is a built-in function. I am fine with this naming.

Reply:
ah I missed that, thanks